lib/matplotlib/markers.py

```diff
@@ -201,7 +201,8 @@ class MarkerStyle:
     # TODO: Is this ever used as a non-constant?
     _point_size_reduction = 0.5
 
-    def __init__(self, marker=None, fillstyle=None):
+    def __init__(self, marker=None, fillstyle=None, *,
+                 normalization="classic"):
```

Comment:
is there a reason this can't be a `transform=` kwarg? ie `transform=None` would do what we did before...